contracts/FlowIDTableStaking.cdc

```diff
@@ -878,13 +841,35 @@ pub contract FlowIDTableStaking {
 
             emit NewDelegatorCutPercentage(newCutPercentage: FlowIDTableStaking.nodeDelegatingRewardCut)
         }
+
+        pub fun setClaimed() {
+
+            let claimedNetAddressDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/networkingAddressesClaimed)
+                ?? panic("Invalid path for networking address dictionary")
+
+            let claimedNetKeyDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/networkingKeysClaimed)
+                ?? panic("Invalid path for networking key dictionary")
+
+            let claimedStakingKeysDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/stakingKeysClaimed)
+                ?? panic("Invalid path for staking key dictionary")
+
+            for nodeID in FlowIDTableStaking.nodes.keys {
+                claimedNetAddressDictionary[FlowIDTableStaking.nodes[nodeID]?.networkingAddress!]
```

Comment:
```suggestion
                claimedNetAddressDictionary[FlowIDTableStaking.nodes[nodeID]?.networkingAddress!] = true
```

Looks like we're not writing anything to the dictionary but might be my Cadence knowledge limitations

Reply:
yeah you're right. I was waiting to add a test until I got the first review, but I'll add one for it now